### PR TITLE
Remove temp profile dir on close

### PR DESCRIPTION
**Pull request by pawrequest**
i cleared out my windows temp dir (appdata/local) a week ago, and today i found ~500 FlaskWebGui dirs, 200k subdirs, ~300k files totalling 30gb. 

so i added a TEMP_DIRS_CREATED global list and cleanup_temp_dirs() function. i append profile dir link to the list when it's defined, then call cleanup_temp_dirs() just before kill_port() on exit

### CODE REVIEW

**Comment by pawrequest** [comment]
i think my ruff profile has made some minor spurious formatting edits, feel free to run your own formatter...

File: src/flaskwebgui.py [hunk 1]
```diff
@@ -11,16 +11,26 @@
...
 
+TEMP_DIRS_CREATED = []
+
+
+def cleanup_temp_dirs():
+    for dir_path in TEMP_DIRS_CREATED:
+        if os.path.exists(dir_path):
+            print(f'Removing Temp dir {dir_path}')
+            shutil.rmtree(dir_path, ignore_errors=True)
+
```
Comment (ClimenteA) [inline src/flaskwebgui.py:33]: No need for list and the cleanup_temp_dirs. 
On each app startup only 1 profile dir for the browser is created in tmp folder.

delete them

File: src/flaskwebgui.py [hunk 2]
```diff
@@ -271,10 +283,12 @@ def start_browser(self, server_process: Union[Thread, Process]):
         if isinstance(server_process, Process):
             if self.on_shutdown is not None:
                 self.on_shutdown()
+            cleanup_temp_dirs()
```
Comment (ClimenteA) [inline src/flaskwebgui.py:286]: Replace `cleanup_temp_dirs()` func with `shutil.rmtree(self.profile_dir, ignore_errors=True)`

File: src/flaskwebgui.py [hunk 3]
```diff
@@ -271,10 +283,12 @@ def start_browser(self, server_process: Union[Thread, Process]):
...
             server_process.kill()
         else:
             if self.on_shutdown is not None:
                 self.on_shutdown()
+            cleanup_temp_dirs()
```
Comment (ClimenteA) [inline src/flaskwebgui.py:291]: Same here:
Replace `cleanup_temp_dirs()` func with `shutil.rmtree(self.profile_dir, ignore_errors=True)`

Make sure to bump version in setup.py file

**Review by ClimenteA — CHANGES_REQUESTED** [review]

**Review by ClimenteA — APPROVED** [review]

**Pull request merged**